### PR TITLE
Hotfix meet bug

### PR DESCRIPTION
**Pull request by shapkarin**
Fix the bug at meet page.
New conference was not inited by the Vue prop change.

### CODE REVIEW

File: src/views/Conference.vue [hunk 1]
```diff
@@ -24,23 +24,24 @@ export default {
   props: {
     room: { type: String, default: '' },
   },
+  watch: {
+    // I don't like this approach
```
Comment (venimus) [inline src/views/Conference.vue:28]: I don't as well

Comment (shapkarin) [reply src/views/Conference.vue:28]: the problem that by the Vue router change and the prop change we don't trigget mounted method as before.. If I can understand why that happen, I can make some better solution.

Comment (shapkarin) [reply src/views/Conference.vue:28]: @venimus we talked and looks like it's a normal Vue flow - don't trigger mounted at props change. So we have two ways: this one, and the second one is create a separate component that will get that prop and change the router.js.

Comment (shapkarin) [reply src/views/Conference.vue:28]: @venimus maybe there is possible to enter the room by jitsi API and remove that all.

Comment (venimus) [reply src/views/Conference.vue]: I was talking about the DOM manipulations actually, but seems you fixed it

**Pull request merged**